Kernel/Bus/PCI/Controller/HostController.h

```diff
@@ -86,6 +86,16 @@ class HostController {
     void enumerate_attached_devices(Function<void(EnumerableDeviceIdentifier const&)> callback, Function<void(EnumerableDeviceIdentifier const&)> post_bridge_callback = nullptr);
     void configure_attached_devices(PCIConfiguration&);
 
+    ErrorOr<PhysicalAddress> translate_bus_address_to_host_address(BARSpaceType, u64) const;
+
+    struct Window {
```

Comment:
I'm not really sure about using the name "Window" here, but that seems to be the term typically used for address translation regions.